components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -360,6 +360,7 @@ struct AccountInfo {
   bool is_imported;
   HardwareInfo? hardware;
   CoinType coin;
+  string keyring_id;
```

Comment:
This should be `string? keyring_id;` and we need a comment saying this is only used for `Filecoin`